lg_media/scripts/image_checker.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python
+import json
+import rospy
+import subprocess
+from lg_common.srv import USCSMessage
+
+PGREP_CMD = ['pgrep', '-a', 'feh']
+EGREP_CMD = ['egrep', '-o', 'http?://[^ ]+']
+PARTIAL_KILLER_CMD = ['pkill', '-9', '-f']
+IMAGE_PROCS_TO_KILL = ['image_viewer.py', 'feh', 'pqiv']
```

Comment:
When you're `pkill -f`ing things, these patterns are potentially a lot more destructive than you intend.  Extending these regex will prevent collateral damage.